library/src/main/java/com/mindera/skeletoid/analytics/Analytics.java

```diff
@@ -98,24 +95,39 @@ public static void removeAllAppenders() {
 
     /**
      * Track Event method - Analytics generic method to send an event with a payload
-     * @param screenName
-     * @param analyticsPayload
+     *
+     * @param screenName       Name of screen
+     * @param analyticsPayload Generic analytics payload
      */
-    public static void trackEvent(String screenName, Map<String, Object> analyticsPayload) {
+    public static void trackEvent(String screenName, Map<String, String> analyticsPayload) {
         getInstance().trackEvent(screenName, analyticsPayload);
     }
 
     /**
      * Track Page Hits - Analytics generic method to track page hits
-     * @param screenName
-     * @param analyticsPayload
+     *
+     * @param screenName       Name of screen
+     * @param analyticsPayload Generic analytics payload
      */
-    public static void trackPageHit(String screenName, Map<String, Object> analyticsPayload) {
+    public static void trackPageHit(String screenName, Map<String, String> analyticsPayload) {
```

Comment:
Please merge this 2 methods into a single on.

Reply:
Done